spec/models/event_schedule_spec.rb

```diff
@@ -17,5 +17,47 @@
     it { is_expected.to validate_presence_of(:event) }
     it { is_expected.to validate_presence_of(:room) }
     it { is_expected.to validate_presence_of(:start_time) }
+
+    describe '#start_after_end_hour' do
+      context 'is invalid' do
+        it 'when event schedule start_time is after the conference end_hour, and returns an error message' do
+          conference = create(:conference)
```

Comment:
move the variables to a `let`, then you don't need to create then same for every test